WooCommerce/Classes/ViewRelated/Upgrades/WooPlanFeatureBenefitsView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import Yosemite
+import WooFoundation
+
+struct WooPlanFeatureBenefitsView: View {
+    let wooPlanFeatureGroup: WooPlanFeatureGroup
+
+    var body: some View {
+        ScrollView(.vertical, showsIndicators: true) {
+            VStack {
+                RoundedRectangle(cornerRadius: Layout.imageCardCornerRadius)
+                    .frame(height: Layout.imageCardHeight)
+                    .foregroundColor(wooPlanFeatureGroup.imageCardColor)
+                    .overlay(
+                        Image(wooPlanFeatureGroup.imageFilename)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .padding(.vertical, Layout.imageCardImageVerticalPadding)
+                    )
+
+                Text(wooPlanFeatureGroup.title)
+                    .font(.subheadline)
+                    .fontWeight(.semibold)
+                    .foregroundColor(.secondary)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+
+                ForEach(wooPlanFeatureGroup.features, id: \.title) { feature in
+                    WooPlanFeatureBenefitRow(feature: feature)
+                }
+
+                Spacer()
+            }
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+            .background(.clear)
+            .padding()
+        }
+        .navigationTitle(wooPlanFeatureGroup.title)
+    }
+
+    private enum Layout {
+        static let imageCardCornerRadius: CGFloat = 16
+        static let imageCardHeight: CGFloat = 164
+        static let imageCardImageVerticalPadding: CGFloat = 22
+    }
+}
+
+struct WooPlanFeatureBenefitRow: View {
+    let feature: WooPlanFeature
+
+    var body: some View {
+        HStack(alignment: .top) {
+            Image(uiImage: .checkmarkImage)
+                .renderingMode(.template)
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: Layout.checkmarkWidth)
+                .foregroundColor(.withColorStudio(name: .green, shade: .shade40))
+
+            VStack(alignment: .leading, spacing: Layout.titleDescriptionVerticalSpacing) {
+                Text(feature.title)
+                    .font(.body)
+
+                Text(feature.description)
+                    .font(.footnote)
+                    .foregroundColor(.secondary)
+            }
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .padding(.vertical, Layout.verticalPadding)
+    }
+
+    private enum Layout {
+        static let verticalPadding: CGFloat = 4
+        static let checkmarkWidth: CGFloat = 24
+        static let titleDescriptionVerticalSpacing: CGFloat = 2
+        static let checkmarkToTextHorizontalSpacing: CGFloat = 8
+    }
+}
+
+struct WooPlanFeatureBenefitsView_Previews: PreviewProvider {
+    static let featureGroup: WooPlanFeatureGroup = WooPlan()!.planFeatureGroups[0]
```

Comment:
We could create some sort of static method or convenience init when we need to use a WooPlan around but we do not care about its details, let's say:

```
public extension WooPlan {
    static func wooPlanFactory() -> WooPlan {
        WooPlan.init(id: "", name: "", shortName: "", planFrequency: .month, planDescription: "", headerImageFileName: "", planFeatureGroups: [])
    }
}
```
```suggestion
    static let featureGroup: WooPlanFeatureGroup = WooPlan.wooPlanFactory().planFeatureGroups[0]
```